sdcm/fill_db_data.py

```diff
@@ -46,7 +49,10 @@ class FillDatabaseData(ClusterTester):
     NULL_VALUES_SUPPORT_ENTERPRISE_MIN_VERSION = "2021.2.dev"
     NEW_SORTING_ORDER_WITH_SECONDARY_INDEXES_OS_MIN_VERSION = "4.4.rc0"
     NEW_SORTING_ORDER_WITH_SECONDARY_INDEXES_ENTERPRISE_MIN_VERSION = "2021.2.dev"
+    CDC_SUPPORT_MIN_VERSION = "4.3"
+    CDC_SUPPORT_MIN_ENTERPRISE_VERSION = "2021"
```

Comment:
```
>>> parse_version('2021') ==  parse_version('2021.0')
True
```

But we don't have 2021.0 release, the code will still work as expected.

We need to test unstable 2021.1, so  we can set  CDC_SUPPORT_MIN_ENTERPRISE_VERSION `2021.1.dev`

parse_version('2021.1.dev') < parse_version('2021.1.rc0')  < parse_version('2021.1.0')

Reply:
Done